dividend_payment.py

```diff
@@ -0,0 +1,12 @@
+class DividendTransaction:
```

Comment:
The file is called `dividend_payment.py`, but the class is called `DividendTransaction`. Let's use the same name so it is not confusing.

Let's change the class to `DividendPayment`.